handlers/bulk_refunds.go

```diff
@@ -0,0 +1,26 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/companieshouse/chs.go/log"
+)
+
+// HandleBulkRefund accepts a bulk refunds file and adds and updates
+// refunds data to the DB
+func HandleBulkRefund(w http.ResponseWriter, req *http.Request) {
```

Comment:
Rename this to make it GovPay specific, we might make it generic for both payment methods when we implement paypal but separate endpoints is the way we're planning on doing it originally